index.html

```diff
@@ -180,5 +180,44 @@ <h1>About me</h1>
                 </ul>
             </div>
         </section>
+
+        <section class="contact-section">
+           <p class="contact-par">I'm always interested in hearing about new projects, so if you'd like to chat get in touch</p>
+            <form>
+                <ul class="first-item-inner-about">
+                    <li>
+                        <li>
+                            <label for="fullname"></label>
+                            <input id="fullname" type="text" name="full_name" placeholder="fullname" size="30" required>
+                        </li>
+                        <li>
+                            <label for="email"></label>
+                            <input id="email" type="email" name="email_name" placeholder="email" size="25" required>
+                        </li>
+                        <li>
+                            <label for="textarea"></label>
+                            <textarea id="textarea" type="textarea" name="textarea" size="500" required>Enter your comment here...</textarea>
+                
+                        </li>
+                       
+                    </li>
+                </ul>
+            <button class="button"> <a type="submit">Get in touch</a></button>
```

Comment:
- You did a great job by adding the submit button but is just visible in the desktop version and is not submit the form. Kindle makes sure the button for submitting the form is visible in the mobile version and working as expected as required by the project. check below your design compared to the Figma design  :grinning: :grinning: :grinning: 

- Button interactions such as hover, active (when it's pressed), disabled, and enable (default state) are styled as shown in the Figma template.

## Color to use for your button 

![image](https://user-images.githubusercontent.com/95297251/192595688-231ed778-ef68-4072-8485-7e35ec9cf2df.png)


## Your design 

![image](https://user-images.githubusercontent.com/95297251/192594495-da5428ab-07d3-4cb2-be13-1a7afd12318c.png)


## Figma design

![image](https://user-images.githubusercontent.com/95297251/192595007-e4431c6e-be68-4b99-a256-0f73dbf6fcc5.png)

Reply:
Done.